requirements/edx-sandbox/py38.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
I think we'll want to only run this job under a 3.8 environment.

Reply:
I've used the [Upgrade Requirements](https://github.com/openedx/edx-platform/actions/workflows/upgrade-python-requirements.yml) Github Action to make a new PR and will be reviewing that one: https://github.com/openedx/edx-platform/pull/30401